Clean Swift/Interactor.xctemplate/___FILEBASENAME___Interactor.swift

```diff
@@ -8,24 +8,13 @@
 
 import UIKit
 
-protocol ___FILEBASENAMEASIDENTIFIER___InteractorOutput {
-    func presentSomething(response: ___FILEBASENAMEASIDENTIFIER___Response)
+protocol ___VARIABLE_sceneName___InteractorOutput {
 }
 
-class ___FILEBASENAMEASIDENTIFIER___Interactor: ___FILEBASENAMEASIDENTIFIER___ViewControllerOutput {
-    var output: ___FILEBASENAMEASIDENTIFIER___InteractorOutput!
-    var worker: ___FILEBASENAMEASIDENTIFIER___WorkerProtocol!
+class ___VARIABLE_sceneName___Interactor: ___VARIABLE_sceneName___ViewControllerOutput {
+    var presenter: ___VARIABLE_sceneName___InteractorOutput!
+    var worker: ___VARIABLE_sceneName___WorkerProtocol! = ___VARIABLE_sceneName___Worker()
```

Comment:
The force-cast is not needed here, isn't it? Even for testing.

Reply:
It actually is needed. Usually, you want to supply your own worker spy that conforms to `___VARIABLE_sceneName___WorkerProtocol` in order to test whether worker's methods are called.